lua/lint/linters/vulture.lua

```diff
@@ -0,0 +1,14 @@
+-- path/to/file:line:col: code message
+local pattern = '([^:]+):(%d+):(.*)'
```

Comment:
If the output is `path/to/file:line:col: code message`, shouldn't there be another pattern and group for the column?

Reply:
This comment comes from another file I used as a template. An example output is the next one, where no column or code is displayed:
```
/home/user/test/test.py:2: unused variable 'i' (60% confidence)
/home/user/test/test.py:3: unused variable 'test_variable' (60% confidence)
```
I updated that comment just in case. I might be adding othger linters I tipically use. Thanks!